src/Miningcore/Payments/PayoutManager.cs

```diff
@@ -62,68 +62,86 @@ public class PayoutManager
         private TimeSpan interval;
         private ClusterConfig clusterConfig;
         private static readonly ILogger logger = LogManager.GetCurrentClassLogger();
+        private static Thread payoutThread;
 
         // Start Payment Services
         public void Start()
         {
-            Task.Run(async () =>
-            {
-                logger.Info(() => "Starting Payout Manager");
+            logger.Info(() => "Starting Payout Manager");
 
-                while(!cts.IsCancellationRequested)
-                {
-                    //try
-                    //{
-                    //await ProcessPoolPaymentsAsync();
-                    foreach(var pool in clusterConfig.Pools.Where(x => x.Enabled && x.PaymentProcessing.Enabled))
-                    {
-                        logger.Info(() => $"Processing payments for pool [{pool.Id}]");
+            // The observable will trigger the observer once every interval
+            Observable.Interval(interval).Subscribe(_ => PayoutCallback());
 
-                        try
-                        {
-                            var handler = await ResolveAndConfigurePayoutHandlerAsync(pool);
+            // Run the initial payout
+            PayoutCallback();
+        }
 
-                            // resolve payout scheme
-                            var scheme = ctx.ResolveKeyed<IPayoutScheme>(pool.PaymentProcessing.PayoutScheme);
+        public void PayoutCallback() 
+        {
+            if (!cts.IsCancellationRequested)
+            {
+                if (null == payoutThread || !payoutThread.IsAlive)
```

Comment:
We are moving away from Task to Thread as opposed to other loops and base code. Just want to highlight so we don't fight against thread handling after this.